### PR TITLE
chore: set up node 16 in workflows

### PR DESCRIPTION
**Pull request by itssharmasandeep**

### CODE REVIEW

File: .github/workflows/build-and-test.yml [hunk 1]
```diff
@@ -11,6 +11,11 @@ jobs:
       - name: Check out code
         uses: actions/checkout@v2
 
+      - name: Set up Node 16
```
Comment (anandtiwary) [inline .github/workflows/build-and-test.yml:14]: Looks fine to me. Or should we use a different build image @aaron-steinfeld?

Comment (aaron-steinfeld) [reply .github/workflows/build-and-test.yml:14]: ubuntu 20.04 is pretty much their newest, I believe https://github.com/actions/virtual-environments#available-environments - and it uses `Npm 6.14.14`, `Node 14.17.5`

Comment (aaron-steinfeld) [reply .github/workflows/build-and-test.yml:14]: (I'm assuming updating node in this way updates npm too)

Comment (anandtiwary) [reply .github/workflows/build-and-test.yml:14]: Yep. it should install npm

**Review by anandtiwary — APPROVED** [review]

File: .github/workflows/build-and-test.yml (hunk 1 above)
Comment (anandtiwary) [reply .github/workflows/build-and-test.yml:14]: going to merge this.

**Pull request merged**